frontend/packages/dev-console/src/components/import/section/FormSection.tsx

```diff
@@ -1,22 +1,40 @@
 import * as React from 'react';
-import FormSectionHeading from './FormSectionHeading';
-import FormSectionDivider from './FormSectionDivider';
-import FormSectionSubHeading from './FormSectionSubHeading';
+import cx from 'classnames';
+import { SectionHeading } from '@console/internal/components/utils';
+import { FormHelperText } from '@patternfly/react-core';
 
 export interface FormSectionProps {
-  title: string;
+  title?: string;
   subTitle?: string;
-  divider?: boolean;
+  badge?: React.ReactNode;
+  fullWidth?: boolean;
   children: React.ReactNode;
 }
 
-const FormSection: React.FC<FormSectionProps> = ({ title, subTitle, divider, children }) => (
-  <React.Fragment>
-    <FormSectionHeading title={title} />
-    {subTitle && <FormSectionSubHeading subTitle={subTitle} />}
+const FormSection: React.FC<FormSectionProps> = ({
+  title,
+  subTitle,
+  fullWidth,
+  badge,
+  children,
+}) => (
+  <div className={cx('pf-c-form', { 'co-m-pane__form': !fullWidth })}>
+    {title && (
+      <SectionHeading
+        text={title}
+        style={{
+          alignItems: 'center',
+          fontWeight: 'var(--pf-global--FontWeight--semi-bold)',
+          justifyContent: 'flex-start',
+          margin: 0,
```

Comment:
Not a big fan of overriding `SectionHeading` and the way the `TechPreviewBadge` is added. We discussed this on slack.